### PR TITLE
Updated to 1.19.4

### PR DESCRIPTION
**Pull request by xpsiggy**
- Packet changed to packet.Packet
- Removed most instances of "int zOffset" or "int z" and replaced with MatrixStack operation
- Reset ShaderColor at end of drawBackground() since it shaded items for some reason
- Commented out DefaultedTranslatableText constructor with "super(key)"
- Added "null" to DefaultedTranslatableText
- Removed z component of drawPosAwareComponent of ScreenMixin class (removed the ' I ' character too)

Have not tested the VanillaPreviewRenderer since that seems to be called due to changed config and I could not get it to work.

### CODE REVIEW

File: common/src/main/java/com/misterpemodder/shulkerboxtooltip/api/renderer/PreviewRenderer.java [hunk 1]
```diff
@@ -95,13 +95,13 @@ static PreviewRenderer getVanillaRendererInstance() {
    *
    * @param x X position of the preview's upper-right corner.
    * @param y Y position of the preview's upper-right corner.
-   * @param z The depth of the preview.
+   * @param z (removed) The depth of the preview.
```
Comment (guillaume-a) [inline common/src/main/java/com/misterpemodder/shulkerboxtooltip/api/renderer/PreviewRenderer.java]: if the var is removes from arguments, shouldn't you delete this comment too ?

File: common/src/main/java/com/misterpemodder/shulkerboxtooltip/impl/renderer/BasePreviewRenderer.java [hunk 2]
```diff
@@ -94,34 +95,34 @@ private void drawItem(ItemStack stack, int x, int y, TextRenderer textRenderer,
...
-    itemRenderer.renderGuiItemOverlay(textRenderer, stack, x, y, countLabel);
+    itemRenderer.renderInGuiWithOverrides(matrices, stack , x, y);
+    itemRenderer.renderGuiItemOverlay(matrices, textRenderer, stack, x, y, countLabel);
   }
 
-  protected void drawItems(int x, int y, int z, TextRenderer textRenderer,
+  protected void drawItems(MatrixStack matrices, int x, int y, TextRenderer textRenderer,
     ItemRenderer itemRenderer) {
-    float prevOffset = itemRenderer.zOffset;
+    // float prevOffset = itemRenderer.zOffset;
```
Comment (guillaume-a) [inline common/src/main/java/com/misterpemodder/shulkerboxtooltip/impl/renderer/BasePreviewRenderer.java]: You should avoid commit dead code. If variable not used anymore, simply delete it

**Comment by R00tB33rMan** [comment]
Reviewed and tested this JAR. No issues/bugs I could identify. Everything appears to be operating accordingly.

**Comment by MisterPeModder** [comment]
I'm sorry I did not notice your PR, I updated the mod to 1.19.4 two days ago by myself.
Thanks a lot for the contribution though, I'll try to be more reactive next time.
The update should be available to download in about an hour (depends on Curse's validation).

**Pull request closed without merging**